pyigm/metallicity/mcmc.py

```diff
@@ -140,6 +201,8 @@ def __init__(self,data,infodata,model,nwalkers,nsamp,threads,outsave,optim,effnh
         self.final=0
         self.optim=optim      #flag to switch type of optimisation
         self.effnhi=effnhi    #flag to use effective NHI
+        self.logUconstraint=str(logUconstraint)
```

Comment:
For annoying "These are passed as input from a text file... So how to compare input `True` and `False` (which are strings) to booleans?" reasons, as discussed in Durham.  This is a kludge, I suppose.